src/binary/YRuby.cc

```diff
@@ -110,6 +110,9 @@ YRuby::~YRuby()
     y2milestone( "Shutting down ruby interpreter." );
     //ruby_finalize(); Do not finalize to allow clear work inside ruby
     _y_ruby_finalized = true;
+    // call at_exit methods, it is part of ruby_finalize, which we have to skip,
+    // so use only needed parts
+    rb_exec_end_proc();
```

Comment:
One of the crashes caused by this: https://ci.opensuse.org/job/yast-yast2-github-push/119//console
@jreidinger so this runs only at the time global objects are destroyed which is too late :(